### PR TITLE
feat(task): allow superusers to view latest data for any task

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: orchestra/utils/task_lifecycle.py [hunk 1]
```diff
@@ -403,12 +403,16 @@ def get_task_overview_for_worker(task_id, worker):
             Information about `task` and its assignment for `worker`.
     """
     task = Task.objects.get(id=task_id)
-    if not task.is_worker_assigned(worker):
-        raise TaskAssignmentError('Worker is not associated with task')
     task_details = get_task_details(task_id)
 
-    task_assignment = TaskAssignment.objects.get(worker=worker,
-                                                 task=task)
+    if task.is_worker_assigned(worker):
```
Comment (marcua) [inline orchestra/utils/task_lifecycle.py:408]: update tests to include superuser

**Review by marcua — CHANGES_REQUESTED** [review]
Quick testing note

File: orchestra/utils/task_lifecycle.py (hunk 1 above)
Comment (jrbotros) [reply orchestra/utils/task_lifecycle.py:408]: ✅

**Review by marcua — APPROVED** [review]
Looks great!

**Pull request merged**